pkg/config/v1alpha1/types.go

```diff
@@ -140,6 +140,10 @@ type ControllerWebhook struct {
 	// must be named tls.key and tls.crt, respectively.
 	// +optional
 	CertDir string `json:"certDir,omitempty"`
+
+	// TLSVersion is the minimum version of TLS supported. Accepts
+	// "", "1.0", "1.1", "1.2" and "1.3" only ("" is equivalent to "1.0" for backwards compatibility)
+	TLSMinVersion string
```

Comment:
I couldn't add TLSOpts because of its type
```bash
hack/tools/bin/controller-gen object paths="./pkg/config/v1alpha1/...;./examples/configfile/custom/v1alpha1/..."
sigs.k8s.io/controller-runtime/pkg/config/v1alpha1:-: name requested for invalid type: func(*crypto/tls.Config)
sigs.k8s.io/controller-runtime/pkg/config/v1alpha1:-: invalid slice element type: func(*crypto/tls.Config)
137:27: missing ',' in parameter list
Error: not all generators ran successfully
```

Not sure if we want to add TLSMinVersion to this type (as I did). Opinions?